### PR TITLE
update layout to post loki styles

### PR DESCRIPTION
**Pull request by btkostner**
Compliment PR to #205. Updates the layout file to post loki styles.

### CODE REVIEW

File: src/houston/public/styles/houston.css [hunk 1]
```diff
@@ -8,37 +8,28 @@
 ******/
 
 nav {
-    background-color: #3a416f;
+    background-color: #403757;
```
Comment (fabianthoma) [inline src/houston/public/styles/houston.css]: on the actual website the navbar has no background color, we should do the same here for the main entry page

Comment (btkostner) [inline src/houston/public/styles/houston.css]: should be fixed now

Comment (lewisgoddard) [inline src/houston/public/styles/houston.css]: You've changed `src/houston/public/styles/index.css` to transparent, but not `src/houston/public/styles/houston.css`

Is that intentional?

Comment (btkostner) [inline src/houston/public/styles/houston.css]: Yes, the `houston.css` file applies to all the pages, which most of them use the colored header. The `index.css` should overwrite that style to match the current website developer page.

**Pull request merged**